src/structures/NewsChannel.js

```diff
@@ -10,6 +10,7 @@ const { Error } = require('../errors');
 class NewsChannel extends BaseGuildTextChannel {
   /**
    * Adds the target to this channel's followers.
+   * <info>If you need the created webhook id, use {@link GuildChannelManager#addFollower}.</info>
```

Comment:
Maybe instead mark this one as deprecated and point to the new method

Reply:
You mean mark the whole method as deprecated? I believe this method will stay as a helper, so I'm not sure that's the best approach.